src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1

```diff
@@ -37,6 +37,10 @@ class AzSKADOServiceMapping: CommandBase
             if((Test-Path $this.BuildMappingsFilePath) -and (Test-Path $this.ReleaseMappingsFilePath))
             {
                 $this.GetBuildReleaseMapping();
+                if ([string]::IsNullOrWhiteSpace($this.MappingType) -or $this.MappingType -eq "All" -or $this.MappingType -eq "Build")
+                {
+                    $this.FetchPipelineConnMapping();
```

Comment:
there are some issue in these two methods FetchPipelineConnMapping and FetchSvcConnMapping why both the methods are generating serviceconnection ST files

Reply:
added for testing, removed extra one